lib/osutil/net.go

```diff
@@ -21,7 +23,7 @@ func GetLans() ([]*net.IPNet, error) {
 		if intf.Flags&net.FlagRunning == 0 {
 			continue
 		}
-		if intf.Flags&net.FlagPointToPoint != 0 {
+		if !includePtP && intf.Flags&net.FlagPointToPoint != 0 {
```

Comment:
Should we use `excludePtP` ? I'd avoid a flag that's only used inverted.

Reply:
Yeah... I though it was nicer to avoid the double negative on the caller side, so we don't have to think "pass false to not exclude PtP"

Reply:
Polluting the caller is worse. Let's stick with `include`.